packages/samples/angular/scripts/link-hoisted-dependencies.js

```diff
@@ -0,0 +1,63 @@
+// TODO: remove this script when nohoist (https://github.com/npm/rfcs/issues/287) is introduced.
```

Comment:
Prior to the package-lock getting updated by the renovate bot, this wasn't necessary, it looks like Angular was correctly linking dependencies in its `node_modules`. Doing a checkout on the package-lock from `master` while keeping the other dependency updates for some reason causes Atomic's `dist` folder to be empty.

I spent a long while trying to find alternative solutions and this solution was my last resort. If we switch to pnpm, yarn or any other package manager, this may also no longer be necessary, so I think it's gonna be a pretty important investigation to do relatively soon.